### ISSUE TITLE
apt-get update/upgrade breaks server

### ISSUE THREAD

**Issue by kyle95wm** [ab77/netflix-proxy#156]
Okay so I've had this issue pop up twice, and each time I had to rebuild my server from scratch.

I run an apt-get update and upgrade, but this time I was asked to replace the docker config file. I said no (because I was under the assumtpion that I'd break something) turns out after a reboot the networking simply stopped working, and now im in the process of rebuilding everything.

Is there a safe way to upgrade my server without it completly blowing up in my face?

**Comment by ab77** [comment]
Don't treat your servers as snowflakes, just destroy and rebuild.

-- ab1

**Comment by kyle95wm** [comment]
Alright I know I should just rebuild but let me ask you this. How easy is it to transfer over everyone's accounts and associated IP's? I have subscribers on GameWisp using this and the last thing I want is to keep saying "please re-authorize" each time I rebuild. Do I just have to copy the auth.db file over? Maybe I should just leave my packages as they are, but thing is I want to either make my server secure by applying the latest patches or make it easy to recover everyone's details.

**Comment by ab77** [comment]
Are you making money off this?

**Comment by kyle95wm** [comment]
Not entirely no but I thought it would be a nice benefit.

**Comment by kyle95wm** [comment]
Regardless if I'm making money off this or not, my question still stands. Should I just rebuild my server if updates are required and also how easy is it to transfer auth info from one install to another?

**Comment by ab77** [comment]
Sorry mate, I don't have time to run a free helpdesk. If you want support, I am sure we can arrange something, otherwise there is always Google.

:grimacing:

**Comment by kyle95wm** [comment]
Hey I don't mean to re-open this on purpose, but I took a snapshot before running updates on my server and same behaviour, HOWEVR when I waited the full 120 seconds my server finally rebooted. It was some "cloud-init" stuff and said something like "Waiting 120 seconds for Network Device" or something like that. Should Ubuntu really be doing this out of the box? My other servers don't do this. It's not impacting how the server works, but 120 seconds to get the server back is kind of unnecessary. I know I should be re-building like you've kept saying, but even when rebooting the droplet without updating it, it shows this symptom.
